varada_trino_manager/commands/call_home.py

```diff
@@ -0,0 +1,18 @@
+from ..infra.call_home_methods import run
+from click import group, argument
+
+@group()
+def call_home():
+    """
+    call home related commands
+    """
+    pass
+
+
+@argument("config_json", nargs=1)
```

Comment:
you can use the option type and then to use click.Path type to assert it exists using the framework